tests/createProjectDriver.py

```diff
@@ -0,0 +1,136 @@
+
+from loginDriver import logIn
+from selenium.webdriver import Chrome
+import time
+
+class createProject:
+
+
+    def __init__(self):
+        self.driver = Chrome()
+    
+    def Close(self):
+        self.driver.quit()
+    
+    def setProjectName(self, projectname, projectpassword):
+        #This is to set up the projectName + description
+        self.driver.find_element_by_id("project_name").send_keys(projectname)
+        self.driver.find_element_by_id("project_description").send_keys(projectpassword)
+    
+    def download(self):
+        self.driver.find_element_by_link_text("(Download a sample roster files)").click()
+        self.driver.implicitly_wait(10)
+    
+    def setRoster(self, studentFile):
+
+        self.driver.find_element_by_id("student_file").send_keys(studentFile)
+    
+    def setRubrics(self, jsonFile):
+        self.driver.find_element_by_id("json_file").send_keys(jsonFile)
+    
+    def driver_createProject(self, username, password, projectname, projectpassword, 
+        studentFile = "C:/Users/Wangj/Downloads/sample_roster.xlsx", jsonFile = "C:/Users/Wangj/Downloads/teamwork_scale3.json"):
+        
+        logIn.Driver_Login(self,username, password) #login first
+        self.driver.implicitly_wait(5)
+        self.driver.execute_script("arguments[0].click()",self.driver.find_element_by_link_text("Create New Project"))        
+        self.driver.implicitly_wait(5)
+
+        createProject.setProjectName(self, projectname, projectpassword)
```

Comment:
all method names should be lower case, with underscores (e.g., set_project_name)

Use the refactoring tools in your IDE to do this automatically.

Reply:
All methods now follow this naming convention.